pysb/examples/michment.py

```diff
@@ -0,0 +1,26 @@
+from pysb import *
```

Comment:
I think when people hear "Michaelis-Menten" they generally think of the M-M equation for the reaction velocity rather than that piecewise mass-action decomposition. Would it be OK if I later revise this to use the actual M-M equation, and create a parallel model that uses the mass-action formulation? We could then have a run script that simulates both over some parameter ranges and plots the differences.